cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -75,8 +75,21 @@ data:
         root /var/www;
         index index.html;
         large_client_header_buffers 4 32k;
-        add_header Cache-Control "max-age=300";
         add_header Cache-Control "must-revalidate";
+{{- if .Values.saml.enabled }}
+        add_header Cache-Control "max-age=0";
+        location / {
+            auth_request /auth;
+            proxy_redirect off;
+            proxy_http_version 1.1;
+            proxy_set_header Connection "";
+            proxy_set_header  X-Real-IP  $remote_addr;
+            proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
+            error_page 401 = /login;
```

Comment:
Do you know if we want or need `403` to take you to `/login` as well?

Reply:
401 deals with authentication and 403 deals with authorization. 403 implies un-authentication but 403 implies un-authorization, 403s should be reserved for fine-grained authorization failing on already logged-in users. so i think just 401s for now.